examples/performance.rs

```diff
@@ -29,10 +31,50 @@ fn performance_with_parameter(param1: String, param2: u32) -> String {
     return format!("Waited {duration:?} for param {param1} - {param2}");
 }
 
+#[get("/performance/skip")]
+fn performance_skipped() -> String {
+    let duration = Duration::from_millis(100);
+    thread::sleep(duration);
+    return format!("Waited {duration:?}\nTransaction will be dropped");
+}
+
+#[get("/performance/rng")]
```

Comment:
Nit: slight preference for "random" over "rng" (which is jargon)

```suggestion
#[get("/performance/random")]
```